em.go

```diff
@@ -275,37 +275,73 @@ func (e *Editor) replaceMacros(text string) string {
 }
 
 func (e *Editor) ReSub(start, end int, cmd rune, text string) {
-    parts := strings.Split(text, "/")
-
-    if len(parts) != 4 {
-        e.Error("no match")
-        return
-    }
+	sep := string(text[1])
```

Comment:
Add a check here to make sure the input isn't just 's'. Currently em crashes if it is.